src/lib.rs

```diff
@@ -1077,6 +1103,38 @@ pub unsafe extern "C" fn blazesym_new() -> *mut blazesym {
     Box::into_raw(c_box)
 }
 
+/// Create an instance of blazesym a symbolizer for C API.
+///
+/// # Safety
+///
+/// Free the pointer with [`blazesym_free()`].
+///
+#[no_mangle]
+pub unsafe extern "C" fn blazesym_new_opt(features: *const blazesym_feature,
```

Comment:
libbpf is using "_opts" suffix, it would be nice to use similar naming here (even though blazesym is completely independent). I find "_opt" a bit confusing because it can be read as "optimized" which is quite a different intent.

Reply:
Fixed it.